index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title> Portfolio </title>
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+ 
+ <div class="container">
+  
+  <div class="top-icons">
+   <div class="left-top"> <span style="font-family: Roboto; font-weight: 500;">9:41</span></div>
+   <div class="right-top">
+   <img src="/images/Bars/Status Bars/Cellular Connection.png">
+   <img src="/images/Bars/Status Bars/Wifi.png">
+   <img src="/images/Bars/Status Bars/Battery.png">
+  </div>
+  </div>
+  <div class="mylogo">
+   <div class="left-logo"> <span style="font-family: Poppins; font-weight: 700; color: #6070FF;">My Logo</span></div>
+   <div class="right-logo">
+    <img src="/images/Bars/Normal Button/Tertiary/Icons/Union.png">
+   </div>
+  </div>
+  <div class="img">
+  <div class="bio">
+   <header class="header">
+    <h1 id="title">I’m Mariane Glad to see you!</h1>
+    <p id="paragraph">I’m a software developer! I can help you build a product , 
+     feature or website Look through some of my work and experience! 
+     If you like what you see and have a project you need coded, 
+     don’t hestiate to contact me.</p>
+     <h3 id="connect">LET’S CONNECT</h3>
+   </header>
+  </div>
+  <div class="bottom-icons">
+   <div class="left-bottom">
+    <img src="/images/social media icons.png">
+   </div>
```

Comment:
- [ ] Please use an unordered list to add the social icons to your document. Your social icons list should look like;
```html
<ul>
  <li>
    <a href="#"><img src="" /></a>
  </li>
  <li>
    <a href="#"><img src="" /></a>
  </li>
...
</ul>
The `<a>` tags are so that you can link the icons to your social media.